js/data/question.jsonc

```diff
@@ -0,0 +1,599 @@
+	"show_additional_stats": "艦隊詳細",
+	"single_fleet": "通常艦隊",
+	"combined_fleet": "連合艦隊",
+	"carrier_task_force": "空母機動部隊",
+	"surface_task_force": "水上打撃部隊",
+	"transport_combined_fleet": "輸送護衛部隊",
+	"strike_force": "遊撃部隊",
+	"set_all_morale_to": "cond値一括適用",
+	
+	"fleet_air_power": "制空値",
+	"fleet_air_power_combined": "艦隊全体",
+	"fleet_air_power_lbas": "vs LBAS",
+	"transport": "TP(S)",
+	"effective_los": "Effective LoS (HQ120)",
+	"shell_power": "砲戦火力",
+	"nb_power": "夜戦火力",
+	"asw_power": "対潜",
+	"can_oasw": "先制対潜",
+	"standard_accuracy": "命中",
+	"fit_accuracy": "フィット砲命中",
+    "fit_accuracy_night": "フィット砲命中(夜戦)",
+	"standard_evasion": "回避",
+	"artillery_spot_rate": "特殊攻撃",
+    "in_ASP_AS": "制空: 確保 / 優勢",
+    "night_attack": "夜戦",
+    "anti_air": "対空",
+    "proportional": "割合撃墜",
+    "flat": "固定撃墜",
+    // The type of anti-aircraft cut-in (#1, #2...) is generally expressed as (1種, 2種...) in Japanese
+	"aaci": "対空CI",
+	"aaci_none": "不発",
+    "aaci_planes": "固定ボーナス",
+    "aaci_rate": "発動率",
+    // For what?
+	"aaci_rate_fleet": "Fleet",
+    "aaci_multiplier": "変動ボーナス",
+    "rocket_barrage": "噴進弾幕発動率",
+    "plane_proficiency": "艦載機熟練度",
+    "prof_acc_bonus": "命中ボーナス",
+    "prof_crit_rate_bonus": "クリティカル率ボーナス",
+    "prof_crit_multiplier": "クリティカル率",
+    "prof_air_power": "制空値加算",
+    "night_contact": "夜間触接率",
+    "sas_improvement": "改修効果",
+	"impr_shelling_power": "砲戦火力",
+	"impr_shelling_acc": "砲戦命中",
+    "impr_shelling_evade": "砲戦回避",
+    "impr_night_power": "夜戦火力",
+    "impr_night_acc": "夜戦命中",
+    "impr_torpedo_power": "雷装",
+    "impr_torpedo_acc": "雷撃命中",
+    "impr_torpedo_evade": "雷撃回避",
+    "impr_asw_power": "対潜",
+    "impr_asw_acc": "対潜命中",
+    "impr_armour": "装甲",
+    "vs_installation_power": "対地火力",
+    "soft_skin": "ソフトスキン",
+    "pillbox": "砲台/トーチカ",
+	"iso_hime": "Iso Hime", // Who?
+    "harbour_summer": "港湾夏姫",
+    "supply_post_mod": "集積地棲姫補正",
+    "anchorage_post_mod": "泊地棲姫補正",
+    "dock_post_mod": "船渠棲姫補正",
+	
+    "sortie_air_power": "制空値(出撃)",
+    "defence_air_power": "制空値(防空)",
+    "high_altitude": "高高度迎撃",
+    "lbas_range": "半径",
+	"lbas_sortie_cost": "出撃コスト",
+    "lbas_fuel": "燃料",
+    "lbas_ammo": "弾薬",
+	
+	"import_export_data_format": "{link_db} と {link_aircalc} から出力されるデータをインポート。",
+	"deckbuilder": "デッキビルダー(本家)",
+	"aerial_combat_simulator": "制空シミュレータ",
+    "note_db_from_text": "メモ: 艦隊編成欄の「デッキビルダーから」にデッキビルダーデータを貼り付けることで、特定の艦隊だけインポートすることもできます。",
+	"lbas": "基地航空隊",
+	"import_support_normal": "道中支援",
+	"import_support_boss": "決戦支援",
+	"export_text": "艦隊&基地航空隊データを出力",
+	"open_in_deckbuilder": "デッキビルダーで開く",
+	"open_in_aircalc": "制空シミュで開く",
+	
+	"world": "海域",
+	"map": "エリア",
+	"edges": "マス",
+    "edges_desc": "例: 1,2,3",
+	"gauge": "攻略段階",
+	"map_hp": "ゲージ残量",
+	"difficulty": "難易度",
+	"diff_all": "全て",
+	"diff_A": "甲",
+	"diff_B": "乙",
+	"diff_C": "丙",
+	"diff_D": "丁",
+	"hq_level": "司令部レベル",
+	"ff_skip_comps": "Skip comps with same ship in Main Fleet?", // ?

```

Comment:
ON = If a Friend Fleet contains a ship which is also currently present in the player's Main Fleet, then do not import this Friend Fleet.